### ISSUE TITLE
Can soot team provide a more understandable tutorial?

### ISSUE THREAD

**Issue by ITeddy** [Sable/soot#773]
I have to say soot is a great tool. But the tutorial is hard to follow. 
Can the develop team provide a more understandable, easy-to-follow, hand-by-hand tutorial? May be with more executable code (not ONE line or several lines of code! I have no idea where to put those codes in) and concrete example.

Thanks in advance

**Comment by ericbodden** [comment]
Hi.

We are trying our best but it’s hard to keep this up to date. Note that we are all doing this in our spare time… Which particular tutorial were you looking at?

Cheers
Eric
